starter-project-backend-g32/src/controllers/commentController.ts

```diff
@@ -0,0 +1,72 @@
+import {Request,Response} from "express";
+import Comment, { IComment } from '../models/Comment'
+
+
+/** returns all comment */
+export const getAllComment = async (req:Request,res:Response) =>{
+    try{
+        const comments:Array<IComment> = await Comment.find({})
+        res.status(200).json(comments)
+    }
+    catch(err){
+        res.status(500).json(err)
+    }
+}
+
+/** returns a comment match given id */
+
+export const getCommentByID = async(req:Request,res:Response)=>{
+    try{
+        const comment:IComment|null = await Comment.findById(req.params.id)
+
+        if (comment != null){
+            res.status(200).json(comment)
+        }
+        else{
+            
+            res.status(204).json({message:"comment not found"})
```

Comment:
make staus code 404 since it's a client error, 204 is for success message in statndard